README_zh-TW.md

```diff
@@ -0,0 +1,431 @@
+# Commit messages guide
+
+[![Say Thanks!](https://img.shields.io/badge/Say%20Thanks-!-1EAEDB.svg)](https://saythanks.io/to/RomuloOliveira)
+
+這是一份了解 commit 訊息的重要性和如何更好的撰寫它的指導手冊。
+
+它可以幫助你了解什麼是 commit、為什麼撰寫好的訊息很重要、最好的實踐案例以及一些技巧來計劃和 (重新) 撰寫良好的 commit 紀錄。
+
+## 什麼是“commit”？
+
+簡單來說，commit 就是在本地資料庫中撰寫的檔案的 _快照_。與印象中不同的是，[git 不僅儲存不同版本檔案之間的差異，還儲存了所有檔案的完整版本](https://git-scm.com/book/eo/v1/Ekkomenci-Git-Basics#Snapshots,-Not-Differences)。對於兩個 commit 之間沒有被修改的檔案，git 只儲存指向前一個完全相同的檔案的鏈結。
+
+下面的圖片展示了 git 如何隨著時間儲存資料，其中每個 “Version” 都是一個 commit：
+
+![](https://i.stack.imgur.com/AQ5TG.png)
+
+## 為什麼 commit 資訊很重要？
+
+- 加快和簡化 code review
+- 幫助理解一個變更
+- 解釋那些無法只透過程式碼來描述的“為什麼”
+- 幫助未來的維護人員明白為什麼以及如何產生的變更，從而使問題排查和除錯更容易
+
+為了最大化這些結果，我們可以使用下一節中描述的一些好的實踐方式和標準。
+
+## 好的實踐方式
+
+這些是從我的經驗、網路文章和其他指導中整理的一些實踐經驗。如果你有更多的經驗（或持不同意見），請隨時提交 Pull Request 來提供幫助。
+
+### 使用祈使句
+
+```
+# Good
+Use InventoryBackendPool to retrieve inventory backend
+用 InventoryBackendPool 獲取庫存
+```
+
+```
+# Bad
+Used InventoryBackendPool to retrieve inventory backend
+InventoryBackendPool 被用於獲取庫存
+```
+
+_不過為什麼要使用祈使句呢？_
+
+commit 訊息描述的效果是這個變更實際上**做**了什麼，而不是被做了什麼。
+
+[Chris Beams 的這篇好文](https://chris.beams.io/posts/git-commit/)為我們提供了一些簡單的句子，可以幫助我們用祈使句寫出更好的 commit 訊息：
+
+```
+If applied, this commit will <commit message>
+如果通過，這個 commit 將會 <commit 訊息>
+```
+
+例子：
+
+```
+# Good
+If applied, this commit will use InventoryBackendPool to retrieve inventory backend
+如果通過，這個 commit 將使用 InventoryBackendPool 獲取庫存
+```
+
+```
+# Bad
+If applied, this commit will used InventoryBackendPool to retrieve inventory backend
+如果通過，InventoryBackendPool 將會被用在獲取庫取
```

Comment:
> 獲取庫取

ditto

Reply:
Fixed in [a1f5756](https://github.com/RomuloOliveira/commit-messages-guide/pull/48/commits/a1f575674b3202623565b80e1705522355b4298d)